src/main/java/com/alkemy/ong/auth/config/SecurityConfiguration.java

```diff
@@ -98,7 +98,7 @@ protected void configure(HttpSecurity httpSecurity) throws Exception {
 
                 //Contacts
                 .antMatchers(POST, "/contacts").hasAuthority("USER")
-
+                .antMatchers(GET, "/contacts/contacts").hasAuthority("ADMIN")
```

Comment:
I think the route just goes with one "/contacts"

Reply:
I did it like this so as not to have a cross with the post method that uses the same endpoit for user and mine is from Admins

Reply:
but it is a Get method shouldn't be a problem

Reply:
change name now